### PR TITLE
Add hierarchical control (FC) and species information (AC)

### PR DESCRIPTION
**Pull request by alan-wu**
fCCB has now approved the changes, this new features are now ready to be merged.

It can be tested here - https://mapcore-demo.org/current/flatmapvuer/

### CODE REVIEW

**Review by Tehsurfer — APPROVED** [review]
The legend works very well!

I didn't hook it up to mapintegratedvuer but I would be interested to see if anything changes, I thought that we use the 'resource-selected' emit to send resources clicked to the sidebar for searches.

Also, currently I append 'species' to the end on the taxonomy when displayed on the prov pop up. I think we want to remove that now? 
image

File: src/App.vue [hunk 1]
```diff
@@ -71,6 +71,7 @@ export default {
       console.log(component);
       let taxon = component.mapImp.describes;
       let id = component.mapImp.addMarker("UBERON:0000948");
+      window.flatmapImp =  component.mapImp;
```
[the remark this answers is not in the document]
Comment (alan-wu) [reply src/App.vue:74]: Good catch.

File: src/components/Tooltip.vue [hunk 2]
```diff
@@ -178,9 +178,6 @@ export default {
     }
   },
   methods: {
-    resourceSelected: function(action) {
```
[the remark this answers is not in the document]
Comment (alan-wu) [reply src/components/Tooltip.vue:181]: No, I do not see how this one is called.

File: src/components/FlatmapVuer.vue [hunk 3]
```diff
@@ -545,11 +631,8 @@ export default {
           "block";
       };
     },
-    resourceSelected: function(action) {
```
[the remark this answers is not in the document]
Comment (alan-wu) [reply src/components/FlatmapVuer.vue:548]: The only caller of this function is the tooltip component but the tooltip component itself never emits the resource-selected to call this.

File: src/App.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (alan-wu) [reply src/App.vue:74]: Oh, this one is fine. It is for the demo app for testing.

[the remark this answers is not in the document]
Comment (Tehsurfer) [reply src/App.vue:74]: yeah true that can be nice to have

[the remark this answers is not in the document]
Comment (Tehsurfer) [reply src/App.vue:74]: yeah true that can be nice to have

**Comment by alan-wu** [comment]
> The legend works very well!
> 
> I didn't hook it up to mapintegratedvuer but I would be interested to see if anything changes, I thought that we use the 'resource-selected' emit to send resources clicked to the sidebar for searches.
> 
> Also, currently I append 'species' to the end on the taxonomy when displayed on the prov pop up. I think we want to remove that now? image

I will leave it like this. I did not receive any feedback from Andre or fCCB regarding this.

**Pull request merged**